src/newsfeed/infrastructure/event_storages.py

```diff
@@ -84,6 +92,108 @@ def __init__(self, config: Dict[str, str]):
             del newsfeed_storage[event_index]
 
 
+class RedisEventStorage(EventStorage):
+    """Event storage that stores events in redis."""
+
+    def __init__(self, config: Dict[str, str]):
+        """Initialize storage."""
+        super().__init__(config)
+
+        redis_config = parse_redis_dsn(config['dsn'])
+        self._pool = aioredis.pool.ConnectionsPool(
+            address=redis_config['address'],
+            db=int(redis_config['db']),
+            create_connection_timeout=int(redis_config['connection_timeout']),
+            minsize=int(redis_config['minsize']),
+            maxsize=int(redis_config['maxsize']),
+            encoding='utf-8',
+        )
+        self._max_newsfeed_ids = int(config['max_newsfeeds'])
```

Comment:
We can remove these two params `max_newsfeed_ids` & `max_events_per_newsfeed_id`.